src/components/GroupConfigCreateEditPage/GroupConfigCreateEditPage.tsx

```diff
@@ -125,6 +129,7 @@ export const GroupConfigCreateEditPage = ({ isEditPage = false }: IGroupConfigCr
         const groupConfig: GroupConfiguration = response.data;
 
         if (groupConfig.productVersion) {
+          setFieldValues({ name: groupConfig.name });
```

Comment:
also please remove setting of `name` in the `setFieldValues` call below since it is set here now

Reply:
<img width="310" alt="image" src="https://github.com/project-ncl/pnc-web-ui-react/assets/6870455/0705ac30-7ddf-4e31-99a2-cf076417eaf9">
unfortunately the `setFieldValues` function, as how it was named, will override any existing value thus causing a result like this. So we have to keep it. This also troubles me a lot because sometimes I do want to change specific fields, are we planning to have `setFieldValue` function for useForm hook in the future?

Reply:
Okay, I did not realize this.
`setFieldValue` might be implemented.